elements/pfe-sass/maps/_general.scss

```diff
@@ -27,6 +27,8 @@ $pfe-vars: (
   icon-size:                     1em,                         //
 
   ui--element--size:             20px,                        //
+  ui--element--size--md:         32px,                        //
```

Comment:
@starryeyez024 Would love your feedback on this!

Reply:
I don't mind the existence of more ui size vars, but I am pondering if they should be base 8 numbers? like 72 instead of 75, 36 instead of 32.

Reply:
yeah, I'm wondering that myself 🤔 Where did the number 75 come from @coreyvickery?

Reply:
I figured this out!  It was a temporary estimate for width of the items and the JS updates this.  Going to leave this hardcoded.